content/roles_and_responsibilities.md

```diff
@@ -1,19 +1,64 @@
+        - The quizmaster is not allowed to say “again” until the quizzer has come to the end of the verse, each time through, as this would give the quizzer information about where their mistake was made.
+    - Once all of the information in both the question and answer have been given, but something needs to be fixed (e.g. Unique word) the quizmaster will say “Again”.
+    - On a Situation (SIT) type question, when the quizzer has completed the Quotation the quizmaster will say, “Quote is complete.”
+    - On an INT or MA Question, if all of the information in the answer has been given, but the quizzer is missing information in the question, the quizmaster will say “More”.
+    - On a Situation question, because the quotation is part of the question, the quizmaster will say “More” when required information from the quotation is missing.
+    - On a Q, Q2, CVR, MACVR, CR, MACR question, if either the chapter or verse number were not completely finished when the quizzer jumped, the quizmaster will say “More”. 
+        - Note: On a CVR/MACVR, CR/MACR Question once the quizzer has provided all the information in both the question and answer the quizmaster will say “What is your question”, if the response to the question supplied by the quizzer is “More”, the quizzer should consider whether the quizmaster had completely pronounced the chapter or verse number. 
+    - On a Reference type question, if the quizzer gives a question that is different than the one on the card, the quizmaster and answer judge must ensure that the question given is the exact same type as the one on the card.
+    - Consult with the answer judge(s) at any time he or she feels is necessary, provided it does not infringe upon the 30 seconds allotted the quizzer.
+    - If an explanation of a ruling is requested by a captain, the quizmaster will ask if any team wants to challenge before the explanation is given.
 
 ### Answer Judges
+- All conferring among quizmaster and answer judge(s) shall be done privately.
+    - It is the answer judge who will determine the accuracy of its content and make a ruling accordingly.
+- Will watch the quizmaster’s questions for accuracy of reading.
+- Ensures the quizzer is recognized by the quizmaster when his/her light comes on.
 
 ### Scorekeepers
+- The scorekeeper will tally and immediately announce when a quizzer has erred out, quizzed out, or fouled out.
+- The scorekeeper will keep record of all time-outs.
+- To determine eligibility for re-substitution, the scorekeeper will keep record of the number of questions a substitute quizzer has remained out. 
+- Either read the score or have the score ready to be read at the beginning of time-outs, when there is no scoreboard.
+- The scorekeeper shall have the score sheets signed by the coaches at the conclusion of each quiz, and report to the statistician.
 
 ### Statisticians
```

Comment:
I don't see anything in the current rule book that should go under this top level definition and requirements of a statistician. There are some specifics related to the Internationals statistician.

Reply:
Might be useful to just describe the job. Although, probably better to include it as a section in the best practices doc.